README.md

```diff
@@ -89,6 +89,16 @@ var opts = {
 }
 
 logger.log('My Sample Log Line', opts);
+
+// or pass any associated  object or a string along all lines of a logger
+// note: it will parse only two levels of the object
+logger.addProperty({'loan':'signing'});
+
+// these lines will be associated with the passed object:
```

Comment:
I'd reverse the language: `// The object added as a property above will be included with the following log lines`